adafruit_ble_radio.py

```diff
@@ -142,19 +142,19 @@ def send_bytes(self, message):
         time.sleep(AD_DURATION)
         self.ble.stop_advertising()
 
-    def receive(self):
+    def receive(self, **args):
```

Comment:
I think it would be helpful to add the named argument to the recieve() function as well instead of using `**args` This would make the code a little bit more self documenting.